tests/Performance/webserver/test_scripts/web.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+"""
+It is used to test the tornado web framework
```

Comment:
You should add some more info here, specifically how to use this test (how to invoke it) and what is supposed to be running (and when the test is considered a success and when not)

Reply:
We have not discussed how to invoke the tests. We need to have some document where we describe what to do for doing performance tests. 
The following command invokes the test: 
multimech-run webserver